src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -0,0 +1,226 @@
+    const newText = _.removeNewline(text);
+    this.ui.input.val(newText);
+    this.ui.spacer.text(newText || ' ');
+
+    this.model.set('name', newText);
+  },
+  templateContext() {
+    return {
+      isNew: this.model.isNew(),
+    };
+  },
+  onDomRefresh() {
+    if (this.model.isNew()) {
+      this.ui.input.focus();
+    }
+  },
+});
+
+const EmailView = View.extend({
+  className: 'pos--relative',
+  template: ClinicianEmailTemplate,
+  behaviors: [InputWatcherBehavior],
+  ui: {
+    input: '.js-input',
+    spacer: '.js-spacer',
+  },
+  onWatchKeydown(evt) {
+    if (evt.which === _.ENTER_KEY) {
+      evt.preventDefault();
+      return;
+    }
+  },
+  onWatchChange(text) {
+    const newText = _.removeNewline(text);
+    this.ui.input.val(newText);
+    this.ui.spacer.text(newText || ' ');
+
+    this.model.set('email', newText);
+  },
+  templateContext() {
+    return {
+      isNew: this.model.isNew(),
+    };
+  },
+  onDomRefresh() {
+    if (this.model.isNew()) {
+      this.ui.input.focus();
+    }
+  },
+});
+
+const SaveView = View.extend({
+  className: 'u-margin--t-8 u-text-align--right',
+  template: hbs`
+    <button class="button--text u-margin--r-4 js-cancel">{{ @intl.admin.sidebar.clinician.clinicianSidebarViews.saveView.cancelBtn }}</button>
+    <button class="button--green js-save">{{ @intl.admin.sidebar.clinician.clinicianSidebarViews.saveView.saveBtn }}</button>
+  `,
+  triggers: {
+    'click .js-cancel': 'cancel',
+    'click .js-save': 'save',
+  },
+});
+
+const DisabledSaveView = View.extend({
+  className: 'u-margin--t-8 u-text-align--right',
+  template: hbs`<button class="button--green" disabled>{{ @intl.admin.sidebar.clinician.clinicianSidebarViews.disabledSaveView.saveBtn }}</button>`,
+});
+
+
+const LayoutView = View.extend({
+  className: 'sidebar flex-region',
+  template: ClinicianSidebarTemplate,
+  triggers: {
+    'click .js-close': 'close',
+    'click .js-menu': 'click:menu',
+  },
+  childViewTriggers: {
+    'save': 'save',
+    'cancel': 'cancel',
+  },
+  regions: {
+    name: '[data-name-region]',
+    email: '[data-email-region]',
+    save: '[data-save-region]',
+    // access: '[data-access-region]',
+    role: '[data-role-region]',
+    // groups: '[data-groups-region]',
+  },
+  onRender() {
+    this.showName();
+    this.showEmail();
+    this.showRole();
+    this.showSave();
+  },
+  showName() {
+    this.showChildView('name', new NameView({ model: this.model }));
+  },
+  showEmail() {
+    this.showChildView('email', new EmailView({ model: this.model }));
+  },
+  showRole() {
+    const isDisabled = this.model.isNew();
+    const roleComponent = new RoleComponent({ model: this.model, state: { isDisabled } });
+
+    this.listenTo(roleComponent, 'change:role', role => {
+      this.model.saveRole(role);
+    });
+
+    this.showChildView('role', roleComponent);
+  },
+  showSave() {
+    if (!this.model.isValid()) return this.showDisabledSave();
```

Comment:
We're probably going to want an email validator for this check, so that a model with an invalid email cannot be saved.